lib/rainforest_cli/test_parser.rb

```diff
@@ -26,8 +26,7 @@ def initialize(file_name)
       @test.browsers = []
     end
 
-    TEST_DATA_FIELDS = [:start_uri, :title, :site_id, :browsers].freeze
-    STEP_DATA_FIELDS = [:redirect].freeze
+    TEST_DATA_FIELDS = [:start_uri, :title, :site_id, :tags, :browsers].freeze
```

Comment:
Yeah. The only time we use `TEST_DATA_FIELDS` alone is when we concat it with `CSV_FIELDS` anyway.